src/SFA.DAS.Campaign.Web/Views/Apprentice/Index.cshtml

```diff
@@ -1,8 +1,68 @@
 ﻿@{
+    Layout = "_Layout-campaign";
     ViewData["Title"] = "Apprentice";
 }
 
-<div style="height: 100px;"></div>
-
-<h1 class="heading-xl">Apprentice</h1>
+<div class="hero-banner">
+    <div class="grid-row">
+        <div class="grid-column-one-half">
+            <div class="hero-heading">
+                <h1 class="heading-xl hero-heading__heading">Ignite your career</h1>
+            </div>
+            <h2 class="heading-s hero-banner__sub-heading">Embrace change and become an apprentice</h2>
+            <p class="hero-banner__copy">Do you want to earn while you learn, be debt free at the end of your training and have valuable hands-on experience? Then enter your postcode or town below and start to fire up your apprenticeship journey.</p>
```

Comment:
Is this functionality for postcode search included in this?

Reply:
Yes if its included as part of MVS